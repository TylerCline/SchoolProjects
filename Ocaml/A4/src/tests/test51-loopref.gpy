def f() : unit {
  let i = ref 10 in
  while !i < 10 {
    let j = ref 20 in
    tt
  };
  tt
}

f()