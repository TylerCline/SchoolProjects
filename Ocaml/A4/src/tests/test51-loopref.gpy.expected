def f() :
 unit {
 ((let i = ((ref (10 : int)) : int ref) in ((while (((! (i : int
 ref) : int) < (10 : int)) : bool) do
 ((let j = ((ref (20 : int)) : int ref) in (tt : unit)) : unit) : unit); (tt : unit) : unit)) : unit)
}
(f() : unit)

